src/test/features/response/routes/statement-of-means/employers.ts

```diff
@@ -97,7 +97,7 @@ describe('Defendant response: Statement of means: employers', () => {
             .expect(res => expect(res).to.be.serverError.withText('Error'))
         })
 
-        it('should return 500 and render error page when cannot retrieve draft', async () => {
+        xit('should return 500 and render error page when cannot retrieve draft', async () => {
```

Comment:
We have to revert it.

Reply:
This should definitely be removed before before merge ⏰

Reply:
yes, I agree

Reply:
done